### DISCUSSION TITLE
Option to always show TOTP code ?

### DISCUSSION THREAD

**Discussion by Vongalin** [keepassxreboot/keepassxc#9169] [Q&A] [answered]
Hi! Is there an option (or planned option) to "always show TOTP code" ? or possibly make the button that displays the code "stick" until pressed again ?

Quite often I'll display the TOTP code and then have to navigate out of the window to do something else. When clicking back into the KeepassXC window, the TOTP display is turned off/hidden.

**Comment by droidmonkey** [comment, marked as answer]
This is a bug
